notebooks/signac_203_Integration_with_MongoDB.ipynb

```diff
@@ -6,6 +6,12 @@
    "source": [
```

Comment:
<p>We should remove this as well IMHO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>